account/manager.go

```diff
@@ -624,9 +702,20 @@ func (m *manager) resumeAccount(ctx context.Context, account *Account, // nolint
 				}
 			}
 
-			acctKey := account.TraderKey.PubKey.SerializeCompressed()
-			contextLabel := fmt.Sprintf(" poold -- "+
-				"AccountCreation(acct_key=%x)", acctKey)
+			txFee, err := m.deriveFeeFromTx(ctx, accountTx)
+			fee := &txFee
```

Comment:
I guess in this case we should just return a pointer to `btcutil.Amount` so we don't need to do this here.